File: Iniciante/URI 1017.hs
import Text.Printf

main :: IO ()
main = do
    tempo' <- getLine
    velocidade' <- getLine

    let tempo = read tempo' :: Double
    let velocidade = read velocidade' :: Double
    let kilometros = tempo * velocidade :: Double
    let media = kilometros / 12 :: Double

    putStrLn $ printf "%.3f" media

File: Iniciante/URI 1017.php
<?php
    $tempo = trim(fgets(STDIN));
    $velocidade = trim(fgets(STDIN));
    $kilometros = $tempo * $velocidade;
    $media = $kilometros / 12;
    $media = number_format($media, 3, ".", "");
    echo "$media\n";
?>